felix/dataplane/linux/int_dataplane.go

```diff
@@ -747,12 +758,14 @@ func NewIntDataplaneDriver(config Config) *InternalDataplane {
 
 		kfb := conntrack.KeyFromBytes
 		vfb := conntrack.ValueFromBytes
+		ctMap := bpfMaps[0].CtMap
 		if config.BPFIpv6Enabled {
 			kfb = conntrack.KeyV6FromBytes
 			vfb = conntrack.ValueV6FromBytes
+			ctMap = bpfMaps[1].CtMap
```

Comment:
i think this is also what you should do for the proxy maps and then instantiate once. You can then group all of the stuff that needs to be instantiated twice for v4 and v6. You will need 2 ct scanners too. Btw it might be good to run the ct scanners specifically in parallel.

Reply:
And you should definitely use the name for the index instead of 0 or 1

Reply:
Also the `bpfproxyOpts` is updated somewhere for v6 already